Mathlib/RingTheory/NonZeroDivisors.lean

```diff
@@ -0,0 +1,189 @@
+/-
+Copyright (c) 2020 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenny Lau, Devon Tuma
+
+! This file was ported from Lean 3 source module ring_theory.non_zero_divisors
+! leanprover-community/mathlib commit 1126441d6bccf98c81214a0780c73d499f6721fe
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.GroupTheory.Submonoid.Operations
+import Mathlib.GroupTheory.Submonoid.Membership
+
+/-!
+# Non-zero divisors
+
+-- porting note: should this instead be called `NonZeroDivisors`? capitalisation looks weird...
```

Comment:
I agree it might look weird, but let's stick to the current naming convention for now.
```suggestion
```